test-projects/social-login/auth/facebook.js

```diff
@@ -0,0 +1,16 @@
+const { FacebookAuthStrategy } = require('@keystone-alpha/passport-auth');
```

Comment:
Could we move all the strategies into a single `strategies.js` file? I think it makes sense in the library for them to be in separate files, but in the context of the example project I think it would be useful to see them all in one place to make it easier to understand how they are all similar/different to each other.